js/src/cli.js

```diff
@@ -33,23 +33,25 @@
 */
 /*jshint strict:false */
 
-var debug = process.env.DEBUG_JSBEAUTIFY || process.env.JSBEAUTIFY_DEBUG ? function() {
+var debug = process.env.DEBUG_JSBEAUTIFY || process.env.JSBEAUTIFY_DEBUG ? function () {
```

Comment:
This is probably why you're getting failures.  Run `make` locally and then be sure to commit any changed files.
The build runs the beautifier on the javascript files in project, enforcing code style.